server/gubled.go

```diff
@@ -212,10 +215,11 @@ func StartService() *service.Service {
 	srv.RegisterModules(4, 3, CreateModules(r)...)
 
 	if err = srv.Start(); err != nil {
+		logger.WithField("error", err.Error()).Error("errors occurred while starting service")
 		if err = srv.Stop(); err != nil {
-			logger.WithError(err).Error("Error when stopping service after Start() failed")
+			logger.WithField("error", err.Error()).Error("errors occured when stopping service after it failed to start")
 		}
-		logger.WithError(err).Fatal("Service could not be started")
+		return nil
```

Comment:
also, I admit to be a bit lazy since I wanted to keep the mocks and tests as they were and not change the func signature to return an error :|